Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG OS_SPECIFIC=
```

Comment:
If possible, it would be good to have all of these files under the `installer` subdirectory to keep things organized

Reply:
Since I am doing `COPY . .` below (https://github.com/PintaProject/Pinta/pull/221/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R30), maybe that's not 100% feasible?
> ```
> ...
> Step 12/15 : COPY .. .
> COPY failed: forbidden path outside the build context: .. ()
> ```

However, I am not sure if `COPY . .` is the correct way to go either (too much cache-busting for _any_ change); but I want to first commit something, then maybe improve it

Reply:
Scratch that; it's actually possible: `.` is relatively to where the `docker build` command is executed

Reply:
This change is coming up, when the other discussion is also improved. Does your request apply to the `debian/` directory as well?

Reply:
Yeah, ideally the `debian` directory could go under there too